build.js

```diff
@@ -42,11 +42,9 @@ new ClosureCompiler([
 	'--warning_level=VERBOSE',
 	'--jscomp_warning=lintChecks',
 	'--compilation_level=ADVANCED',
-	'--isolation_mode=IIFE',
-	'--js=src/trace.js',
-	'--js=src/nulldownloader.js',
-	'--js=src/nullzip.js',
-	'--js=src/nullxlsx.js',
+	'--module_resolution=NODE',
+	'--output_wrapper=(function(){%output%}).call(window);',
```

Comment:
Do we need the wrapper for ES module ?

Reply:
There are two ways to import this package:
* `<script src="/path/to/dist/nullxlsx.js">`
This is the script compiled and minified with closure-compiler which has the wrapper. When evaluated, it registers NullXlsx to window.

* `import {NullXlsx} from '@neovici/nullxlsx'`
This imports the raw ES source code. 

So, we need the wrapper to output the minified, register-to-window build.